### ISSUE TITLE
Disable Output Window when debugging

### ISSUE THREAD

**Issue by jasc2v8** [mark-wiemer-org/ahkpp#70]
Please add "Disable Output Window".
This will prevent the Output Window from opening upon an AHK run error.
The user can always Ctrl+Shift+U to open the output window.
Thank you.

**Comment by mark-wiemer** [comment]
This shouldn't be too tough to implement, but it's not the highest priority. Just posting an update here as #492 was just opened as a dupe. This is still very possible and in-scope, but with ~40 issues backlogged and a personal vacation coming up, I can't guarantee it'll be solved this month. I'll be working on reordering the backlog found at [the AHK++ project](https://github.com/orgs/mark-wiemer-org/projects/1/views/1) if you'd like to see where my priorities lie :)

**Comment by mark-wiemer** [comment]
Ref https://github.com/microsoft/vscode/issues/105270, but I do plan to fix this in the upcoming release. Everything's working now, just trying to add automated tests. They're the first of their kind, so it's taking some time :)

**Comment by mark-wiemer** [comment]
OK, further testing shows that even removing any application code to open the debug console view still results on it opening the very first time a debug session begins. Closing the debug console view and debugging again does not reopen the view. Reopening this to see if I can identify the exact issue here, but it'll have to be good enough for a while given the longstanding issues with VS Code itself and the lack of documentation and API support around these features, from what I've been able to find.

**Comment by andyfangaf** [comment]
Will this prevent the output window from popping up by another conflict in an extension?

**Comment by mark-wiemer** [comment]
@andyfangaf no, it shouldn't affect the behavior of other extensions. I'm still researching, but it looks like creating a debug session for the first time is the only case where the output window currently pops up (AHK++ 6.1.1).

I need to rebuild the debugger for advanced use-cases anyway, so I'll be testing a very basic debugger to see if behavior like this is just built-in to VS Code or can be avoided. Stay tuned :)
